docs/use/rest-api.rst

```diff
@@ -1,35 +1,241 @@
+| PUT    | /api/v2/tasks/<task_id>/notes/<note_id>             |                                          |
++--------+-----------------------------------------------------+------------------------------------------+
+| Edit a notesing the HTTP parameter ``text=...``                                                         |
++--------+-----------------------------------------------------+------------------------------------------+
+| DELETE | /api/v2/tasks/<task_id>/notes/<note_id>             | Confirmation message                     |
++--------+-----------------------------------------------------+------------------------------------------+
+| Delete a note                                                                                           |
++--------+-----------------------------------------------------+------------------------------------------+
+
+
+Modules/Other
+-------------
+
++--------+-----------------------------------------------------+------------------------------------------+
+| Method | URI                                                 | Response type                            |
++========+=====================================================+==========================================+
+| GET    | /api/v2/modules                                     | JSON object with module names as keys    |
++--------+-----------------------------------------------------+------------------------------------------+
+| Receive an object whose keys are the names of available of modules. The corresponding value of each key |
+| is a ``true``/``false`` boolean that indicates whether the module is currently activated or not.        |
++--------+-----------------------------------------------------+------------------------------------------+
+|| GET   | /api/v2/analytics/ssdeep_compare                    | Confirmation message                     |
++--------+-----------------------------------------------------+------------------------------------------+
+| Run ssdeep.compare analytic                                                                             |
++--------+-----------------------------------------------------+------------------------------------------+
+| GET    | /api/v2/analytics/ssdeep_group                      | List of lists of SHA256 hash strings     |
++--------+-----------------------------------------------------+------------------------------------------+
+| Receive list of sample hashes grouped into lists by ssdeep hash                                         |
++--------+-----------------------------------------------------+------------------------------------------+
+| GET    | /                                                   | Test response object                     |
++--------+-----------------------------------------------------+------------------------------------------+
+| Test functionality. Should produce: ``{'Message': 'True'}``                                             |
```

Comment:
Tiny thing, but I don't think `True` is quoted anymore. To be consistent with other rows, should the `Response Type` cell for this be `Confirmation message`?